shared/chat/inbox/row/big-team-channel/index.js

```diff
@@ -84,57 +74,75 @@ const mutedStyle = {
 }
 
 const UnreadIcon = () => (
-  <Box style={unreadContainerStyle}>
-    <Box style={unreadStyle} />
+  <Box style={styles.unreadContainer}>
+    <Box style={styles.unread} />
   </Box>
 )
 
-const unreadContainerStyle = {
-  ...globalStyles.flexBoxRow,
-  alignItems: 'center',
-  alignSelf: 'stretch',
-  flex: 1,
-  justifyContent: 'flex-end',
-}
-const unreadStyle = {
-  backgroundColor: globalColors.orange,
-  borderRadius: 6,
-  flexShrink: 0,
-  height: 8,
-  width: 8,
-}
-
-const teamRowContainerStyle = {
-  ...globalStyles.flexBoxRow,
-  ...desktopStyles.clickable,
-  alignItems: 'center',
-  flexShrink: 0,
-  maxHeight: isMobile ? globalMargins.large : globalMargins.medium,
-  minHeight: isMobile ? globalMargins.large : globalMargins.medium,
-  paddingLeft: globalMargins.tiny,
-  paddingRight: isMobile ? globalMargins.tiny : globalMargins.xtiny,
-}
-
-const channelRowContainerStyle = {
-  ...teamRowContainerStyle,
-  alignItems: 'stretch',
-  paddingRight: 0,
-}
-
-const channelBackgroundStyle = {
-  ...globalStyles.flexBoxRow,
-  ...(isMobile ? globalStyles.fillAbsolute : {width: '100%'}),
-  alignItems: 'center',
-  marginLeft: globalMargins.large,
-  paddingLeft: globalMargins.tiny,
-  paddingRight: globalMargins.tiny,
-}
-
-const selectedChannelBackgroundStyle = {
-  ...channelBackgroundStyle,
-  backgroundColor: globalColors.blue,
-  borderTopLeftRadius: 3,
-  borderBottomLeftRadius: 3,
-}
+const styles = styleSheetCreate({
```

Comment:
What's with `styleSheetCreate`? Is this a preferred practice to the method of creating separate objects for each component's styles?

Reply:
Nojima can explain this better, but basically it's a helper for creating cross-platform styles because creating a stylesheet differs between React and React Native. Take a look at the differences between them to understand a little more:

- React: https://github.com/keybase/client/blob/master/shared/styles/index.desktop.js#L113
- React Native:https://github.com/keybase/client/blob/master/shared/styles/index.native.js#L52